src/components/About/Blog/index.js

```diff
@@ -305,140 +305,144 @@ class Blog extends Component {
         {this.state.postRendered && (
           <div>
             <Container>
-              {this.state.posts.map((posts, i) => {
-                const description = htmlToText
-                  .fromString(posts.description)
-                  .split('…');
-                const content = posts.content;
-                let category = [];
-                posts.categories.forEach(cat => {
-                  let k = 0;
-                  for (k = 0; k < allCategories.length; k++) {
-                    if (cat === allCategories[k]) {
-                      category.push(cat);
+              {this.state.posts &&
+                this.state.posts.length > 0 &&
+                Array.isArray(this.state.posts) &&
+                this.state.posts.map((posts, i) => {
```

Comment:
If `this.state.posts` isn't an array, then `this.state.posts.length ` will fail and break the code. Add isArray check before checking length.

Reply:
OKay

Reply:
Done Sir!